gazebo_ros/CMakeLists.txt

```diff
@@ -38,6 +48,7 @@ target_include_directories(gazebo_ros_node
     $<INSTALL_INTERFACE:include>
     ${gazebo_dev_INCLUDE_DIRS}
 )
+target_compile_definitions(gazebo_ros_node PRIVATE "GAZEBO_ROS_NODE_BUILDING_DLL")
```

Comment:
Shouldn't these only be done `if(WIN32)`?

Reply:
Added!